api/v1beta1/hyperconverged_types.go

```diff
@@ -395,6 +395,8 @@ type VirtualMachineOptions struct {
 type HyperConvergedFeatureGates struct {
 	// Allow to expose a limited set of host metrics to guests.
 	// +optional
+	// +kubebuilder:default=false
```

Comment:
@jcanocan please also add the default value for the `FeatureGates` field (line 69), and for the `spec` field (line 824).

Reply:
Done.